src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,39 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  if (!tabs.some(tab => tab.id === activeTabId)) {
+    // eslint-disable-next-line no-param-reassign
```

Comment:
```suggestion
 
```
Consider enabling linter. You can use find with the logical operator OR to find the active tab or assign the tabs[0] as active tab